lib/pbio/include/pbdrv/compiler.h

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025 The Pybricks Authors
+
+#ifndef _PBDRV_COMPILER_H_
+
+// Marks a switch case that intentionally falls through to the next one
+#define FALL_THROUGH                __attribute__((fallthrough))
```

Comment:
```suggestion
#define PBDRV_FALL_THROUGH                __attribute__((fallthrough))
```

everything needs pbio prefix to avoid name clash with other libraries